webui/react/src/utils/data.ts

```diff
@@ -137,8 +130,8 @@ export const deletePathList = (obj: RawJson, path: string[]): void => {
 // We avoid exporting this type to discourage/disallow usage of any.
 /* eslint-disable-next-line @typescript-eslint/no-explicit-any */
 type Mapper = (x: any) => any;
-export const applyMappers = <T>(data: unknown, mappers: Mapper | Mapper[]): T => {
-  let currentData = clone(data);
+export const applyMappers = <T>(data: T, mappers: Mapper | Mapper[]): T => {
```

Comment:
not in scope for this pr: this type annotation doesn't make sense -- if `mapper` returns a different type, the annotation says it returns `T`. Ideally, we'd type the list mapper type such that:

- input types are right: (i.e. `Mapper<T, U>` takes in `T`, returns `U`
- types between mappers are right (i.e. `[Mapper<T, U>, Mapper<U, V>, Mapper<V, W>]`)